nano/node/bootstrap/bootstrap_bulk_pull.cpp

```diff
@@ -1,3 +1,4 @@
+#include <nano/node/bootstrap/block_deserializer.hpp>
```

Comment:
It is not easy to review the changes in this file because it is not easily obvious to me how the internal operation bulk_pull_client. This is a good opportunity to add some comments to explain what it does. For exmaple, it would be good to comment the following variables:
```
	nano::block_hash expected;
	nano::account known_account;
	nano::pull_info pull;
	uint64_t pull_blocks;
	uint64_t unexpected_count;
```